src/serialize/flatten.js

```diff
@@ -0,0 +1,30 @@
+const { flatten } = require("flatley");
+const { cloneDeep } = require("lodash");
+
+const hasPeriodInKey = (key, _) => {
```

Comment:
- why _ ?
- it's a bit weird to return false or throw

Reply:
_ because unused
returns false so that it doesn't expect the transform function
throws error so that it aborts